tasks/main.yml

```diff
@@ -1 +1,62 @@
-# Defaults tasks for role sync2git
+---
+# https://github.com/siteshwar/centos-sync-packages/blob/d24dfb53ce26ac3415d57961664f86484a17735b/setup.sh
+- name: "Setup system to sync packages from RHEL to CentOS"
```

Comment:
You can get rid of all specific items that apply to a playbook , but not for a tasks file included in a role.
And indent back task to beginning of file